chopper/lib/src/utils.dart

```diff
@@ -44,7 +44,7 @@ Request applyHeaders(
 
   for (var k in headers.keys) {
     if (!override && h.containsKey(k)) continue;
-    h[k] = headers[k];
+    h[k] = headers[k]!;
```

Comment:
Is the `!` necessary since `headers` is not nullable (an other curiosity question :D )

Reply:
It looks really interesting, but despite headers are Map<String, String>, heders[k] is String?.  And I have next error message

A value of type 'String?' can't be assigned to a variable of type 'String'.
Try changing the type of the variable, or casting the right-hand type to 'String'.

Reply:
ok so maybe we should do something like this

```dart
var val = headers[k];
if (val == null) continue;
h[k] = val;
```

Reply:
@Vovanella95 can you apply suggested change here ?
To be sure we don't put null value in `Map<String, String> h`